ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -300,6 +301,23 @@ public boolean onTouchEvent(MotionEvent ev) {
     return super.onTouchEvent(ev);
   }
 
+  @Override
+  public boolean executeKeyEvent(KeyEvent event) {
+    int eventKeyCode = event.getKeyCode();
+    if (!mScrollEnabled
+    && (eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN_LEFT
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN_RIGHT
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_UP
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_UP_LEFT
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_UP_RIGHT
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_LEFT
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_RIGHT)) {
+      return false;
+    }
+    return super.executeKeyEvent(event);
+  }
+
```

Comment:
Could you please properly format the code? The logical OR operator must be on the same line and the indentations are missing.

Reply:
@makovkastar I pushed the new modifications, please take a look and tell me if it's Ok for you now.

Reply:
@makovkastar I updated to fit your last comments